lib/js/_create.js

```diff
@@ -76,7 +76,8 @@ module.exports = (defaultConfig, currentWorkingDirectory, allToolkitsInfo) => {
 			// Generate task list for building the new package
 			const pathToPackage = path.resolve(currentWorkingDirectory, toolkitDetails.path);
 			const tasks = new Listr(
-				buildNewPackage(toolkitDetails.config, pathToPackage, globalLicense, allAnswers)
+				buildNewPackage(toolkitDetails.config, pathToPackage, globalLicense, allAnswers),
+				{concurrent: true}
```

Comment:
This will run all tasks in parallel for speed, but where tasks have been made synchronous, they will be bumped to the top of the chain